README.md

```diff
@@ -30,6 +30,7 @@ This Awesome List is maintained by [@BretFisher](https://github.com/BretFisher)
 
 ## Recent News and Updates
 
+- 2024-12 - [Mirantis - Mirantis guarantees long-term support for Swarm](https://www.mirantis.com/blog/mirantis-guarantees-long-term-support-for-swarm/)
```

Comment:
1. The YYYY-MM date is incorrect, please re-read the blog post and find the publish date to ensure this change is using the correct date.
2. Please ensure the title mentions the year that Mirantis guarantees support 'til. It's in the blog post.

Reply:
OK I gave you access to the URL, please go look for the correct publishing date. You only need to change the date from 2024-12 to the one found on the blog post. Then this PR is ready for review.

Reply:
OK I gave you access to the URL, please go look for the correct publishing date. You only need to change the date from 2024-12 to the one found on the blog post. Then this PR is ready for review.